bin/cron/deploy_to_test

```diff
@@ -22,13 +22,17 @@ def main
 
   # TODO(asher): Add a reviewer (namely the next DOTD), add appropriate tags
   # (particularly deploy).
-  pr_number = GitHub.create_and_merge_pull_request(
+  pr_number = GitHub.find_or_create_pull_request(
     base: 'test',
     head: 'staging',
     title: 'DTT (Staging > Test) [robo-dtt]'
   )
   if pr_number.nil?
-    raise Exception.new('GitHub.create_and_merge_pull_request failed.')
+    raise Exception.new('GitHub.find_or_create_pull_request failed.')
+  end
+  
+  unless GitHub.merge_pull_request(pr_number)
+    raise Exception.new('GitHub.merge_pull_request Failed to merge pull request.')
```

Comment:
Similar to my previous suggestion ... most engineers aren't familiar with the details of our complex, custom build pipeline. It might be useful to add more context error message. Maybe: "Failed to merge Pull Request #{pr_number}"